components/eamxx/src/share/util/eamxx_time_interpolation.cpp

```diff
@@ -81,23 +81,28 @@ void TimeInterpolation::perform_time_interpolation(const TimeStamp& time_in)
  *   None
  */
 void TimeInterpolation::add_field(const Field& field_in, const bool store_shallow_copy)
+{
+  // Typical case, name of field matches the name in the data files.
+  add_field(field_in,"",store_shallow_copy);
+}
+void TimeInterpolation::add_field(const Field& field_in, const std::string& alt_name, const bool store_shallow_copy)
```

Comment:
I don't know if we should put the "alt_name" logic in this class. I think the user could easily do a shallow copy of the field, to alias it.
```
my_time_interp.add_field (f1);
my_time_interp.add_field (f2.alias("f2_new_name"));
```
It seems like an unnecessary detail, that time interp need not to know/accommodate.